packages/jest-snapshot/package.json

```diff
@@ -33,14 +36,23 @@
     "semver": "^6.3.0"
   },
   "devDependencies": {
-    "@babel/traverse": "^7.3.4",
+    "@babel/preset-flow": "^7.7.2",
+    "@babel/preset-react": "^7.7.2",
     "@types/natural-compare": "^1.4.0",
     "@types/semver": "^6.0.1",
     "ansi-regex": "^5.0.0",
     "ansi-styles": "^4.2.0",
     "jest-haste-map": "^25.3.0",
     "prettier": "^1.13.4"
   },
+  "peerDependencies": {
+    "@babel/core": "7.x"
+  },
+  "peerDependenciesMeta": {
+    "@babel/core": {
+      "optional": true
```

Comment:
this requires a relatively new version of `npm` or `yarn` to suppress the warning - maybe fine? Probably 🙂

Reply:
I think so yeah. Also, if you install `jest` will it ever even warn? Because you'll have a matching `@babel/core` anyway right

Reply:
If using babel-jest, yeah